packages/test/local-server-stress-tests/src/stressDataObject.ts

```diff
@@ -256,6 +270,25 @@ export class DefaultStressDataObject extends StressDataObject {
 		}
 		this._locallyCreatedObjects.push(obj);
 	}
+
+	private stageControls: StageControls | undefined;
+	public enterStagingMode() {
+		this.stageControls = this.context.containerRuntime.enterStagingMode();
+	}
+
+	public inStagingMode(): boolean {
+		return this.context.containerRuntime.inStagingMode;
+	}
+
+	public exitStagingMode(accept: boolean) {
```

Comment:
Nit: minimize terminology used
```suggestion
	public exitStagingMode(commit: boolean) {
```